### ISSUE TITLE
Create a querying interface

### ISSUE THREAD

**Issue by thomastkok** [thomastkok/multimedia-retrieval#16]
Implement a simple interface for querying:

1. user picks a shape in the database (or alternatively can load a 3D mesh file containing a shape which is not part of the database; note that you can do this easily by simply selecting several shapes, from the benchmarks mentioned in Step 2, and not including them in the database)
2. extract features of the query shape (if not from the database)
3. compute distance of query shape to all database shapes (depends on #15)
4. sort distances from low to high
5. present the K best-matching shapes, plus their distances to the query, in some visual way so the user can inspect them. Alternative: set a distance threshold t and present all shapes in the database having a distance to the query lower than t.
